### ISSUE TITLE
i18n support for rollup docs

### ISSUE THREAD

**Issue by exrem** [rollup/rollup#5582]
### Documentation is

- [ ] Missing
- [ ] Needed
- [ ] Confusing
- [X] Not sure?

### Please Explain in Detail...

hai! ^^

rollup has been getting more popular over time, however, its docs is only available in english and chinese, and it's not that easy to translate. imho, it'd be better if rollup was available in more languages.

### Your Proposal for Changes

all of the translations of the docs can be integrated into the current monorepo (the /docs directory) and internationalization can be made easier using [the native i18n feature of vitepress](https://vitepress.dev/guide/i18n). if you'd like this to be applied, i could contribute w/ the best i can :3

**Comment by lukastaegert** [comment]
That would be nice, but that would just mean that we have a lot of unmaintained translations in the main repository that nobody updates. We had this for years, with the current approach we have at least one maintained translation. If you would be willing on another translation, we can definitely set things up like for the Chinese translation. For that one, there is actually a team maintaining it.

**Comment by exrem** [comment]
aight, i get the reason of the current approach for i18n and totally appreciate it ^^

anyways, my initial purpose was translating the project into azerbaijani & turkish, so, i think i can just fork [rollup-docs-cn](https://github.com/rollup/rollup-docs-cn) and start working on them, right?

**Comment by lukastaegert** [comment]
If you want, we can set up something similar to what we have for the Chinese docs. To my understanding, there is also some GitHub actions to help keep the project up to date if the original changes.

**Comment by exrem** [comment]
yes, please, it would be awesome if you could set such a thing up for az.rollupjs.org and tr.rollupjs.org. thank you so much for the help! ^^

**Comment by lukastaegert** [comment]
Ok, as a first step I cloned the rollup repository twice. For each of the new repositories, you should have full admin rights. Also, I created a team to maintain translations for each language. You should also be admin of those teams so that you can add additional people to support you.
This should allow you to start translating. Once this has reached a sufficiently advanced stage, this is what should happen next:
- Have a look at what the Chinese Docs team did. They added/changed some Github Actions workflows.
- Also, there is the @docschina-bot they set up. Despite its name, this bot is used by similar project setups (e.g. Vite, sli.dev) to manage all other translations. Every time Rollup master is updated, it will create a pull request with the docs changes (in English). The idea is that then you can modify this PR to the correct language and merge it and that way always stay up to date. I do not know how it is set up correctly, but the Chinese Docs team can surely help you.
- We need to set up deployment. This can probably be done via Vercel, I will need to have a look
- We need to set up the sub-domains. Again, not sure if this can be done via Vercel. Maybe also Rich Harris himself needs to do that. We will see.
- When everything is in place, we can update Rollup docs to link to the new translations.

**Comment by exrem** [comment]
thank you so much for the assistance you've provided, sir! i will start working on the translations asap and will do my best for rollup. take care, and have a good day! (＾▽＾)
